pkg/util/deployment/deployment.go

```diff
@@ -28,8 +28,13 @@ import (
 	podutil "k8s.io/kubernetes/pkg/util/pod"
 )
 
+const (
+	// The revision annotation of a deployment's replication controllers which records its rollout sequence
+	RevisionAnnotation = "deployment.kubernetes.io/revision"
```

Comment:
I thought we were grouping deployment (and any kind of resource for that matter) related annotations in the api files (https://github.com/kubernetes/kubernetes/blob/63ec304e42c4f1bb6bd90d3f638cf1aafafe4064/pkg/apis/extensions/types.go#L238)

@bgrant0607